src/main/java/com/faforever/client/mod/ModServiceImpl.java

```diff
@@ -176,7 +176,11 @@ public void loadInstalledMods() {
 
   @Override
   public CompletableFuture<Void> downloadAndInstallMod(String uid) {
-    return fafService.getMod(uid).thenAccept(mod -> downloadAndInstallMod(mod, null, null));
+    return fafService.getMod(uid).thenAccept(mod -> downloadAndInstallMod(mod, null, null))
+        .exceptionally(throwable -> {
+          logger.warn("Mod for modupdate not found", throwable);
+          return null;
+        });
```

Comment:
Whoever calls this method will now know that it failed (only the first `exceptionally` is ever called) so the game will start even though the mod could not be installed.

The error must, therefore, be handled in the calling code.
Also, the assumption "not found" isn't correct as there can be other reasons why an exception was thrown, like disconnect, server down, disk space full, file system permission, out of memory, and so on.